lib/diagnostic.js

```diff
@@ -39,24 +57,35 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = new Pet()
+pet.name = 'Barbra Ann'
+pet.species = 'kitty'
+pet.breed = 'Tortoiseshell'
+pet.vocalization = 'muh muh muh muh muh muh meow'
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
-};
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name
+  this.species = species
+  this.breed = breed
+  this.vocalization = vocalization
+}
 
 // 5. Attach a `vocalize` method to the Pet's prototype.  The `vocalize` method
 // should return the pet's vocalization attribute.
 
-/* your answer here */
+Pet.prototype.vocalize = function () {
+  let noise = this.vocalization
+  }
+  return noise
+}
```

Comment:
Nice! Although you don't have to define a new variable called `noise`, you can simply have your function return `this.vocalization`